vmc/sync_utils.go

```diff
@@ -0,0 +1,100 @@
+/* Copyright 2022 VMware, Inc.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package vmc
```

Comment:
As a general Golang rule, avoid `util` and `helper` naming, see https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Sounds like this file provides tasks related methods?